File: TypeScript/codewars/Sum-of-the-first-nth-term-of-Series.ts
export function SeriesSum(n: number): string {
	let sum = 1;
	let term = 1;

	if (n <= 0) return (0).toFixed(2);

	for (let index = 1; index < n; index++) {
		term = term + 3;
		sum += 1 / term;
	}
	return sum.toFixed(2);
}

console.log(SeriesSum(1));

File: Csharp/CodeWars/src/Resolutions/SumOfTheFirstNthTermOfSeries.cs
namespace CodeWars.src.Resolutions;

public class SumOfTheFirstNthTermOfSeries
{
    public static string SeriesSum(int n)
    {
        double sum = 1;
        double term = 1;

        if (n <= 0) return "0.00";

        for (int index = 1; index < n; index++)
        {
            term += 3;
            sum += 1 / term;
        }
        return $"{sum:F2}";
    }
}
